sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -782,6 +780,20 @@ UUID saveUncaughtException(Thread thread, Throwable throwable, com.microsoft.azu
     }
 
     /**
+     * Interface to use a template method to build exception model since it's a complex operation
+     * and should be called only after all enabled checks have been done.
+     */
+    private interface GetExceptionModel {
```

Comment:
I get confused of this interface name. This is a kind of a data model but actually a wrapper. Can we just say `ExceptionWrapper` or something else?

Reply:
It's used also for native java.